Tropos/Controllers/TRApplicationController.h

```diff
@@ -13,4 +13,6 @@
 - (RACSignal *)localWeatherNotification;
 - (void)setMinimumBackgroundFetchIntervalForApplication:(UIApplication *)application;
 
+- (void)subscribeForNotificationsWithDeviceToken:(NSData *)deviceToken;
```

Comment:
I think this should read `subscribeToNotifications`: generally you subscribe "to" something, not "for" it.